inference-engine/ie_bridges/python/wheel/CMakeLists.txt

```diff
@@ -53,8 +53,12 @@ foreach(_target ov_runtime_libraries ie_plugins _pyngraph pyopenvino)
     endif()
 endforeach()
 
-set(cp_python "cp${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
-set(openvino_wheel_name "openvino-${WHEEL_VERSION}-${WHEEL_BUILD}-${cp_python}-${cp_python}-${WHEEL_PLATFORM}.whl")
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "import wheel.bdist_wheel ; print(f'{wheel.bdist_wheel.get_abi_tag()}')" OUTPUT_VARIABLE PYTHON_ABI)
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "import wheel.vendored.packaging.tags as tags ; print(f'{tags.interpreter_name()}{tags.interpreter_version()}')" OUTPUT_VARIABLE INTERPRETER)
```

Comment:
does it provide correct values during cross-compilation on ARM?

Reply:
In general target python interpreter could be different from those is used on a host for cross-compilation. For now this CMakeList does not provide an option to build for ARM:

https://github.com/openvinotoolkit/openvino/blob/d3cf7b92d0c1aff3b162717fbd7d27741ae9bb08/inference-engine/ie_bridges/python/wheel/CMakeLists.txt#L25

if(APPLE)
    set(WHEEL_PLATFORM macosx_10_15_x86_64)
elseif(UNIX)
    set(WHEEL_PLATFORM manylinux2014_x86_64)
elseif(WIN32)
    set(WHEEL_PLATFORM win_amd64)
    set(TBB_LIBS_DIR runtime/3rdparty/tbb/bin)
else()
    message(FATAL_ERROR "This platform is not supported")
endif()